<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoiseGenerator</class>
 <widget class="QWidget" name="NoiseGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>375</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Noise Generator</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>103</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </widget>
  <widget class="QLabel" name="mapNoise">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>0</y>
     <width>375</width>
     <height>375</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="bilinearButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Bilinear</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="smoothstepButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>122</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Smoothstep</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="bicubicButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Bicubic</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="seedValue">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="QLabel" name="seedLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Seed</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sizeLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Size</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="sizeValue">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>2048</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="QLabel" name="scaleLabel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="scaleValue">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>2048</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="adaptiveCheck">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>275</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="adaptiveLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Adaptive scale</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
